filament/src/fg/FrameGraph.h

```diff
@@ -288,10 +288,8 @@ class FrameGraph {
 
     FrameGraphHandle createResourceNode(fg::ResourceEntryBase* resource) noexcept;
 
-    enum class DiscardPhase { START, END };
-    backend::TargetBufferFlags computeDiscardFlags(DiscardPhase phase,
-            fg::PassNode const* curr, fg::PassNode const* first,
-            fg::RenderTarget const& renderTarget);
+    bool computeDiscard(const Vector<FrameGraphHandle> fg::PassNode::* list,
```

Comment:
This method is answering a yes or no question, "canDiscard" might be a better name, alternatively add a one-line comment.